### ISSUE TITLE
Add illumos support

### ISSUE THREAD

**Issue by papertigers** [sharkdp/bat#476]
Bump the version of console (if its not a point release) once illumos support is merged into `termios-rs`.
https://github.com/dcuddeback/termios-rs/pull/21

**Comment by sharkdp** [comment]
Thank you for the feedback. I'm happy to do this, but I'd prefer to be reminded again once this is actually possible (PR is merged, new version of termios-rs is out, new version of console is out). Please feel free to re-open the ticket then.

**Comment by papertigers** [comment]
That PR was finally merged. 
Is it possible to bump the Cargo.lock so we pick up a console with termios 0.3.2? 

Thanks again!

**Comment by sharkdp** [comment]
Done. Thank you for the reminder!
